### ISSUE TITLE
[Feature Request] About gyro output on switch pro controller...

### ISSUE THREAD

**Issue by RuafuMan-Obs** [Ryujinx/Ryujinx#4760]
### Overview

The camera movement always has a  lag when controller move fastly ,when I steady it the camera drift slowly until the visual  reach the angle controller did,this problem also happens on Betterjoy for Cemu when it turns the pro controller's gyro output to cemuhook;
I test the same controller on Cemu 1.26+(support pro controller motion natively without betterjoy),Yuzu and Citra and their motion camera movement doesn't have this problem at all

### Smaller details

The camera movement lags badly when controller turn up and down ，unconspicuous when turn left and right, Whatever I change the gyro deadzone or sensitivity,the problem remains .now I have to use my phone emulating motion to get accurate gyro output

### Nature of request

pro controller gyro output optimize

### Why would this feature be useful?

Make pro controller to get accurate motion output,which other emulators already have

**Comment by Rarurin** [comment]
Just reporting in that im having the same/similar problem, using original Pro Controller. Gyro will keep drifting to the right infinitely, even when upping the deadzone. In my case i had to put the offset on 8.0 so it would stop, but then you have an obviously malfunctioning experience, and it will be noticeable harder to steer it left. This behavior isn't happening in any other app, like CEMU, Steam, etc. Tested with no Steam running or any other app other than Ryujinx alone.

**Comment by InkblotDisciple** [comment]
On a side note, 8 BitDO Ultimate gyro works perfectly, whereas I've had all above mentioned issues on my Pro Controller

**Comment by anjulalk** [comment]
Yes, Yuzu works perfectly with motion. But in Ryujinx my gyro input always drifts to the right. Even after using BetterJoyForCemu the effect is still there.

**Comment by leocg** [comment]
I have the same issue. Motion work fine with Dualshock 4 but drift to the right with Switch Pro Controller.

**Comment by boymanguy** [comment]
Hi, I have a gulikit kingkongpro 2 and am having the same issue with gyro/motion drift. When using ryujinx, the gyro drifts slowly to the left. I tried adding a deadzone which mitigates the issue but the controller still turns more left than right while aiming in totk. If I move the controller up it still drifts left. It feels like I’m fighting with the controls when trying to aim. I have recalibrated the gyro on the controller many times but that is not the issue. This doesn’t happen for me in yuzu, cemu, or dolphin- only in ryujinx.

**Comment by MrDrumble** [comment]
Still happening. Using an 8bit Do Ultimate. Works fine in other software.

**Comment by markandgo** [comment]
I have a PS4 controller and turning on and off the controller fixes the gyro drift. Drifting didn't happen in the dead emulator not to be named.

**Comment by jxnding** [comment]
I have a GuliKit KK Pro 2 and am getting drift to the right. With a Dualsense 5 it works fine.

**Comment by yogithesymbian** [comment]
> I have a GuliKit KK Pro 2 and am getting drift to the right. With a Dualsense 5 it works fine. 

did you solve ?
